bin/configs/spring-mvc.yaml

```diff
@@ -10,3 +10,4 @@ additionalProperties:
   hideGenerationTimestamp: "true"
   serverPort: "8002"
   additionalModelTypeAnnotations: '@com.fasterxml.jackson.annotation.JsonFilter(value = "filter-name");@com.fasterxml.jackson.annotation.JsonIgnoreProperties(value = "id")'
+  additionalEnumTypeAnnotations: '@com.fasterxml.jackson.annotation.JsonTypeName("myName")'
```

Comment:
Seems to me the JsonTypeName is hardcoded to "myName" for all Enum models in the auto-generated code. What if the JsonTypeName is different for each Enum model?

Reply:
@wing328 this is just an example of usage/verify everything works. This feature is about adding common annotations to all enums

Reply:
@wing328 changed to `@com.fasterxml.jackson.annotation.JsonFormat`